packages/components/src/Tag/Tag.tsx

```diff
@@ -0,0 +1,187 @@
+                case "validationNegative":
+                  return (
+                    <span className={styles.validationIcon}>
+                      <ExclamationIcon role="presentation" />
+                    </span>
+                  )
+                case "validationCautionary":
+                  return (
+                    <span className={styles.validationIcon}>
+                      <CautionIcon role="presentation" />
+                    </span>
+                  )
+                case "validationInformative":
+                  return (
+                    <span className={styles.validationIcon}>
+                      <InformationIcon role="presentation" />
+                    </span>
+                  )
+                case "profile":
+                  return (
+                    <span className={styles.profile}>
+                      {props.avatar && renderAvatar(props.avatar)}
+                    </span>
+                  )
+                default:
+                  return
+              }
+            })()}
+          <span
+            className={classnames(
+              styles.textContent,
+              isTruncated && styles.truncate
+            )}
+            style={{
+              maxWidth: isTruncated ? truncateWidth : undefined,
+            }}
+          >
+            {children}
+          </span>
+          {dismissible && (
+            <>
+              <button
+                type="button"
+                className={styles.dismissButton}
+                onClick={onDismiss}
+                onMouseDown={onMouseDown}
+                onMouseLeave={onMouseLeave}
+              >
+                <div className={styles.iconWrapper}>
+                  <ClearIcon inheritSize role="img" aria-label="Dismiss" />
+                </div>
+              </button>
+            </>
+          )}
+          {variant === "statusLive" && (
+            <span className={styles.liveIcon}>
+              <LiveIcon
+                role="presentation"
+                classNameOverride={styles.liveIcon_base}
+                width="16"
+                height="16"
+                viewBox="0 0 16 16"
+                fill="none"
+              />
+              <LiveIcon
+                role="presentation"
+                classNameOverride={styles.liveIcon_1}
+                width="16"
+                height="16"
+                viewBox="0 0 16 16"
+                fill="none"
+              />
+              <LiveIcon
+                role="presentation"
+                classNameOverride={styles.liveIcon_2}
+                width="16"
+                height="16"
+                viewBox="0 0 16 16"
+                fill="none"
+              />
+              <LiveIcon
+                role="presentation"
+                classNameOverride={styles.liveIcon_3}
+                width="16"
+                height="16"
+                viewBox="0 0 16 16"
+                fill="none"
+              />
```

Comment:
This should be consistent whenever we use the `LiveIcon` right? It should be baked into `LiveIcon` instead (maybe with an `isAnimated` prop if we want the animation to be optional).

Reply:
I opted not to try to change this in the original icon cause its the only one that requires these extra props, and I'd rather consider a change in the SVG/design than to modify our template to accommodate.